docs/source/escape-hatches/distributing-javascript.rst

```diff
@@ -181,7 +44,7 @@ where you can then load any of its exports:
 
 
 Distributing Javascript via PyPI_
```

Comment:
In this section there is a misspelling of `applciation`->`application`, but it is not a new code modification so I cannot comment on it.